bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml

```diff
@@ -4,3 +4,7 @@
 #   - spec_talents=talent_name_d:1/talent_name_e:0/talent_name_f:3
 # Cool Other Build:
 #   - talents=WOW_EXPORT_STRING
+M+ Build:
+   - talents=BsbBnWR3R1Z1HmbUKIpvm3XXhTDCAAAAAAKaIJikEJkEJlkk0CSBJJhkEIJJSESSC
+Cleave Build:
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKapIJikEJIRSLJJNi0CJSIJSQSiEhkkA
```

Comment:
Talked with Preheat and got this talent string for a Cleave build instead: BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKSiIJkQSkUSS0gUCJSIJEkkIRikkA

could you update your PR using this one?